stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -202,21 +192,13 @@ subroutine stochastic_physics_wrapper (GFS_Control, GFS_Data, Atm_block, ierr)
                     write(6,*) 'call to GFS_apply_lndp failed'
                     return
              endif
-             deallocate(stype)
-             deallocate(sfc_wts)
              do nb=1,Atm_block%nblks
                  GFS_Data(nb)%Sfcprop%smc(:,:) =  smc(nb,1:GFS_Control%blksz(nb),:)
                  GFS_Data(nb)%Sfcprop%slc(:,:) =  slc(nb,1:GFS_Control%blksz(nb),:)
                  GFS_Data(nb)%Sfcprop%stc(:,:) =  stc(nb,1:GFS_Control%blksz(nb),:)
                  GFS_Data(nb)%Sfcprop%vfrac(:) =  vfrac(nb,1:GFS_Control%blksz(nb))
              enddo
-             deallocate(smc)
-             deallocate(slc)
-             deallocate(stc)
-             deallocate(vfrac)
          endif ! lndp block
-         deallocate(xlat)
-         deallocate(xlon)
```

Comment:
Where did all the deallocate statements go? This should happen at some point, probably in a "finalize" call to the stochastic physics wrapper. I didn't review the rest of the changes yet, just a quick comment on those deallocate statements.

Reply:
I did removed them, and since there is no finalize as far as I can tell, I didn't know where to put them.

Reply:
I think we should add this phase to the wrapper, and the call to it in `subroutine atmos_model_end` in `atmos_model.F90`.

Reply:
ok. i will add that.